deploy/helm/scf/values.yaml

```diff
@@ -39,3 +39,11 @@ k8s-host-url: ""
 k8s-service-token: ""
 k8s-service-username: ""
 k8s-node-ca: ""
+
+nats:
```

Comment:
Shouldn't this section go under `features` ?

Reply:
Unfortunately, no; because of how helm subcharts work, I can _only_ put it under `nats` (so that the subchart picks up the settings from here).

Basically, the _only_ way to propagate the configs down is to have a top-level key with the subchart name…

Supposedly helm v3 is going to support lua scripting so that can be done, but that isn't implemented yet (even in `3.0.0-beta.3`).  Also, for some insane reason they plan to run subchart scripts before parent chart scripts…

Reply:
Ah! Ok, then I would request a (small) comment about this.